src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaTestHelper.java

```diff
@@ -56,6 +56,31 @@
     return matches;
   }
 
+  public static List<String> getAllLinesFromLog(String logName) {
+    try {
+      return Files.readAllLines(Paths.get(getLogFilePath(logName)));
+    } catch (IOException e) {
```

Comment:
nit: we can merge all exception using pipe, as we are only printing stacktrace

Reply:
done